conf/scalardb-graphql-custom-values.yaml

```diff
@@ -41,31 +41,35 @@ grafanaDashboard:
 
 tolerations:
   - effect: NoSchedule
-    key: kubernetes.io/app
+    key: scalar-labs.com/dedicated-node
     operator: Equal
-    value: scalardbgraphqlpool
+    value: scalardb
```

Comment:
I think that we expect one `Scalar DB Server` and one `Scalar DB GraphQL Server` to be deployed on the one worker node.
So, I set the same tolerations as Scalar DB Server in the custom values file of Scalar DB GraphQL Server.